### ISSUE TITLE
[MODULE] - Workday classifier

### ISSUE THREAD

**Issue by LeonardPuettmann** [code-kern-ai/bricks#161]
**Please describe the module you would like to add to bricks**
Given a specific date, I want to classify if it's a working day or a holiday

**Do you already have an implementation?**
```python
from datetime import date 
from workalendar.usa import UnitedStates

# Get all holidays in the US

US_cal = UnitedStates()
US_cal.holidays(2022)
```

**Additional context**

**Comment by LeonardPuettmannKern** [comment]
Implemented with #264

**Comment by jhoetter** [comment]
@LeonardPuettmannKern I think this one isn't accessible via the UI?
<img width="1440" alt="image" src="https://github.com/code-kern-ai/bricks/assets/62521561/443d0d19-fa8c-4033-afc5-ead6a3996f93">

**Comment by LeonardPuettmannKern** [comment]
@jhoetter Correct. Opened a hotfix PR for it. 👍

**Comment by LeonardPuettmannKern** [comment]
The ATTRIBUTE and COUNTRY_CODE in the refinery code is incorrect and needs to be adjusted. :) https://bricks.kern.ai/classifiers/863

**Comment by LeonardPuettmannKern** [comment]
Fixed with #319
